### DISCUSSION TITLE
How to exclude node_modules from filter search?

### DISCUSSION THREAD

**Discussion by osenvosem** [nvim-neo-tree/neo-tree.nvim#1511] [Q&A]
I have both readme.md placed in root and node_modules in the .gitignore file. Filter doesn't find the readme.md in the root but it finds all such filed in the node_modules folder. That's illogical. How to fix that?

**Comment by madelaney** [comment]
If you're using the `/` command to search, then you should have to so something like:

(shortened for brevity)
```lua
require('neo-tree').setup({
  filtered_items = {
    visible = false,
    hide_gitignored = true,
  }       
})
```

**Reply by osenvosem** [reply]
I already had this

**Comment by osenvosem** [comment]
In order to illustrate the issue

<img width="346" alt="Screenshot 2024-07-24 at 10 55 13" src="https://github.com/user-attachments/assets/70996f62-065e-41f4-b548-1bb03674c002">

How to make the plugin to not search in ignored node_modules folder?
